tests/test_core_to_contrib.py

```diff
@@ -16,787 +16,814 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
 import importlib
-from unittest import TestCase
+from typing import Any
+from unittest import TestCase, mock
 
 from parameterized import parameterized
 
+NEW_HOOK_PATHS = [
+    (
+        "airflow.gcp.hooks.compute.ComputeEngineHook",
+        "airflow.contrib.hooks.gcp_compute_hook.GceHook",
+    ),
+]
+OLD_HOOKS_PATHS = [
```

Comment:
What is the difference between this variable and the above?

Reply:
OLD_HOOKS_PATHS is variable which store paths that are not testable right now, NEW_HOOK_PATH is a place where I put paths that can be tested. With the end of the task OLD_HOOKS_PATHS will be removed permanently and NEW_HOOK_PATH will be replace with HOOK_PATH (just a var name for now), which will store all testable paths.